build.gradle

```diff
@@ -6,7 +6,7 @@ allprojects {
 
   version = "1.0.0"
 
-  sourceCompatibility = 1.7
+  sourceCompatibility = 1.8
```

Comment:
@mpetazzoni to support java 1.8 we need to change this. If we want to make a pull request then it might create compatibility issue, what's your thought?

Reply:
@mpetazzoni Yes I am using java 8 specific, as you suggested to use 
```java
_metricMap.computeIfAbsent(key, n -> createMetric(attributeValue));
```
It give me this error if I use 1.7 
```shell
/Users/rohitvyavahare/work/kafka-monitor/src/main/java/com/linkedin/kmf/services/SignalFxMetricsReporterService.java:158: error: lambda expressions are not supported in -source 1.7
    SettableDoubleGauge metric = _metricMap.computeIfAbsent(key, n -> createMetric(attributeValue));
                                                                   ^
  (use -source 8 or higher to enable lambda expressions)
```